### DISCUSSION TITLE
[Feb 05] sso

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1257] [Discord Threads]
> **SirFixALot** _queried at_ ```2022-02-04T13:26:45.799000+00:00```  
## Is there a cloud version of nocodb where we can use SSO?  
  
> **o1lab** _commented at_ ```2022-02-04T13:27:05.886000+00:00```  
#### not yet  
  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/939149988406104084)  
 
 - - - -

#### Join [NocoDB's](http://www.nocodb.com/)  community
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by dhaavu** [comment]
Do we know when this feature will be available ?
